docs/topics_5/installing-web-console-on-openshift.adoc

```diff
@@ -62,11 +62,22 @@ Two templates are provided, one for shared storage and one without shared storag
 . Click the *{ProductName}* template.
 . Click *Instantiate Template*.
 endif::[]
-ifdef::ocp-45,ocp-41[]
+ifdef::ocp-45[]
 . Review the application settings and click *Create*.
 . In the *Topology* view, wait for the pods to start.
 . Click the *Open URL* button of the `mta` Pod to open the {WebName} in a new browser window.
 endif::[]
+ifdef::ocp-41[]
+. Review the application settings and click *Create*.
+. In the *Topology* view, wait for the pods to start.
+. Click the *Open URL* button of the `mta` Pod to open the {WebName} in a new browser window.
++
+[NOTE]
+====
+The instructions above point your browser to an HTTP address that does not support viewing static reports.
```

Comment:
http or https allow to view exactly the same. 
choosing one or the other is only a matter of user choice, with the same results , but different security